meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/uz3eg-iocc/pm_cfg_obj.c

```diff
@@ -3,6 +3,7 @@
 * SPDX-License-Identifier: MIT
 ******************************************************************************/
 
+
```

Comment:
Please remove extra newline

Reply:
Oh, this is just a file copy from the new pmu-firmware, didn't want to change what is coming from the xilinx tooling.

Basically I just built the new pmu firmware and copied over the new files.